zmon-controller-app/src/main/resources/templates/index.html

```diff
@@ -73,6 +73,33 @@
     <script th:src="${staticUrl} + '/lib/highlightjs/highlight.pack.min.js?t=@buildTime@'" chartset="utf-8"></script>
     <script th:src="${staticUrl} + '/lib/angular-highlightjs/angular-highlightjs.min.js?t=@buildTime@'" chartset="utf-8"></script>
 
+    <!-- AppDynamics JS Agent -->
+    <script charset='UTF-8' th:inline="javascript" th:if="${appdynamicsEnabled}" >
+        var appdynamicsConfig = /*[[${appdynamicsConfig}]]*/ { apiKey: "", adrumExtUrlHttp: "", adrumExtUrlHttps: "", beaconUrlHttp: "", beaconUrlHttps: "", xd: true}
+        console.log('AppDynamics Agent is Enabled', appdynamicsConfig)
```

Comment:
I thought it could be useful in production for a short time, just in case something goes wrong. On a second thought, exposing the apiKey maybe shouldn't be allowed at all?